spec/openapi.yaml

```diff
@@ -7919,14 +8104,16 @@ components:
         created_timestamp: 2019-11-03T11:49:02.807Z
         modified_timestamp: 2019-11-04T11:49:02.807Z
     format:
-      title: format
-      enum:
-      - MMS
-      - SMS
-      - TTS
-      type: string
+      title: MessageType
+      type: array
+      items:
+        type: string
+        enum:
```

Comment:
@isthiiii should `CHANNEL` be added to this list of values?

Reply:
Hey @SamRussellMM,
Checked this with @isthiiii and yes we should add `CHANNEL` here please 🙂